pymc/smc/smc.py

```diff
@@ -41,9 +41,9 @@
 
 
 class SMC_KERNEL(ABC):
-    """Base class for the Sequential Monte Carlo kernels
+    """Base class for the Sequential Monte Carlo kernels.
 
-    To create a new SMC kernel you should subclass from this.
+    To create a new kernel you should subclass from this.
```

Comment:
Keep the SMC here.

Reply:
Thanks! Got this started and opened a new issue. #6214